src/EnergyPlus/SystemReports.cc

```diff
@@ -1053,65 +1053,40 @@ void InitEnergyReports(EnergyPlusData &state)
-                }
-            }
-        }
-        auto &vrpds = state.dataPlnt->VentRepPlantDemandSide(PlantLoopNum);
-        for (BranchNum = 1; BranchNum <= vrpds.TotalBranches; ++BranchNum) {
-            auto &vrpdsBranch = vrpds.Branch(BranchNum);
-            for (CompNum = 1; CompNum <= vrpdsBranch.TotalComponents; ++CompNum) {
-                auto &vrpdsBranchComp = vrpdsBranch.Comp(CompNum);
-                for (VarNum = 1; VarNum <= vrpdsBranchComp.NumMeteredVars; ++VarNum) {
-                    auto &vrpdsBranchCompMeter = vrpdsBranchComp.MeteredVar(VarNum);
-                    VarType = vrpdsBranchCompMeter.ReportVarType;
-                    VarIndex = vrpdsBranchCompMeter.ReportVarIndex;
-                    vrpdsBranchCompMeter.CurMeterReading = GetInternalVariableValue(state, VarType, VarIndex);
+    for (LoopSideLocation LoopSide : DataPlant::LoopSideKeys) {
+        auto &vrp = state.dataPlnt->VentRepPlant[static_cast<int>(LoopSide)](PlantLoopNum);
+        for (PlantLoopNum = 1; PlantLoopNum <= state.dataHVACGlobal->NumPlantLoops; ++PlantLoopNum) {
+            for (BranchNum = 1; BranchNum <= vrp.TotalBranches; ++BranchNum) {
+                auto &vrpBranch = vrp.Branch(BranchNum);
+                for (CompNum = 1; CompNum <= vrpBranch.TotalComponents; ++CompNum) {
+                    auto &vrpBranchComp = vrpBranch.Comp(CompNum);
+                    for (VarNum = 1; VarNum <= vrpBranchComp.NumMeteredVars; ++VarNum) {
+                        auto &vrpBranchCompMeter = vrpBranchComp.MeteredVar(VarNum);
+                        VarType = vrpBranchCompMeter.ReportVarType;
+                        VarIndex = vrpBranchCompMeter.ReportVarIndex;
+                        vrpBranchCompMeter.CurMeterReading = GetInternalVariableValue(state, VarType, VarIndex);
+                    }
                 }
             }
         }
-    }
 
-    // On every iteration, load the Condenser Supply Side Data and load the Condenser Demand Side Data
-    for (PlantLoopNum = 1; PlantLoopNum <= state.dataHVACGlobal->NumCondLoops; ++PlantLoopNum) {
-        auto &vrcss = state.dataPlnt->VentRepCondSupplySide(PlantLoopNum);
-        for (BranchNum = 1; BranchNum <= vrcss.TotalBranches; ++BranchNum) {
-            auto &vrcssBranch = vrcss.Branch(BranchNum);
-            for (CompNum = 1; CompNum <= vrcssBranch.TotalComponents; ++CompNum) {
-                auto &vrcssBranchComp = vrcssBranch.Comp(CompNum);
-                for (VarNum = 1; VarNum <= vrcssBranchComp.NumMeteredVars; ++VarNum) {
-                    auto &vrcssBranchCompMeter = vrcssBranchComp.MeteredVar(VarNum);
-                    VarType = vrcssBranchCompMeter.ReportVarType;
-                    VarIndex = vrcssBranchCompMeter.ReportVarIndex;
-                    vrcssBranchCompMeter.CurMeterReading = GetInternalVariableValue(state, VarType, VarIndex);
-                }
-            }
-        }
-        auto &vrcds = state.dataPlnt->VentRepCondSupplySide(PlantLoopNum);
```

Comment:
Found and fixed bug. Should have been `auto &vrcds = state.dataPlnt->VentRepCondDemandSide(PlantLoopNum);`